docker/docker-compose.yml

```diff
@@ -23,11 +25,11 @@ services:
     depends_on:
     - db
     volumes:
-#    - .:/code
+    - ..:/code
```

Comment:
Was macht das genau mit dem zwei Punkten?

Reply:
das mountet das MOS repo live in den container hinein, dadurch kann man live entwickeln während der Container läuft, ohne, dass man den container neubauen muss.